src/components/AnnotationCounter.vue

```diff
@@ -1,16 +1,16 @@
 <template>
   <div class="main-column">
-    <h3>Annotations</h3>
+    <h3>{{$t('label.annotations')}}</h3>
     <p>
-      Remaining:
+      {{$t('label.remaining_annotations')}}:
```

Comment:
it's better to add the ':' to the string to translate, as there may be a space needed after the word ("Remaining:" in english but "Restantes :" in French

Reply:
Thanks! didn't know that :)